clang/test/Sema/warn-thread-safety-analysis.c

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsyntax-only -verify -Wthread-safety -Wthread-safety-beta %s
+// RUN: %clang_cc1 -fsyntax-only -verify -Wthread-safety -Wthread-safety-beta -fexperimental-late-parse-attributes %s
```

Comment:
@pdherbemont We should probably have a test case that runs without `-fexperimental-late-parse-attributes` so that we check normal parsing still works, especially given that its the default.

Reply:
done!